docs/src/main/asciidoc/rest-json-guide.adoc

```diff
@@ -484,6 +484,20 @@ If you want to use the `keycloak` Quarkus extension in your project, you should
 configuration. See link:keycloak-guide#configuring-cors[the Keycloak guide] for more details.
 ====
 
+== GZip Support
+Quarkus comes with GZip support (even though it is not enabled by default). The following configuration knobs allow to configure GZip support.
+
+[source, properties]
+----
+quarkus.resteasy.gzip.enabled=true // <1>
+quarkus.resteasy.gzip.max-input=10M // <2>
+----
+
+<1> Enable Gzip support.
+<2> Configure the upper limit on deflated request body. The default value is `10M`.
```

Comment:
Can you explain why one would put a upper limit? Why would I use a given property is important.

Reply:
A little explanation added. WDYT?